Common/Transforms/itkAdvancedImageMomentsCalculator.hxx

```diff
@@ -46,7 +46,7 @@ AdvancedImageMomentsCalculator<TImage>::AdvancedImageMomentsCalculator()
 
   /** Threading related variables. */
   this->m_UseMultiThread = true;
-  this->m_Threader = ThreaderType::New();
+  this->m_Threader = MultiThreaderBase::New();
```

Comment:
Might use thread pools, so might improve the performance of AdvancedImageMomentsCalculator  👍